src/view.tree.js

```diff
@@ -99,25 +99,12 @@ class TreeView {
   }
 
   _onItemClick(event) {
-    let $target = $(event.target);
-    let download = false;
```

Comment:
Beside from adding a new method to detect the click target.
I removed this download part, which view the file's raw. So as not to conflict with #892